packages/@webex/plugin-meetings/src/member/types.ts

```diff
@@ -23,6 +23,14 @@ export type ParticipantWithRoles = {
   };
 };
 
+export type ParticipantWithBrb = {
+  controls?: {
+    brb?: {
+      enabled: boolean;
+    };
+  };
+};
```

Comment:
I'm just looking at `ParticipantWithRoles`... in that type, neither `controls` nor `role` is optional, so I just want to double check that we do want `controls` and `brb` to both be optional here.

Reply:
@brycetham I couldn't find information in the locus dto documents, so I will remove the optional parameter since none of my test cases require it to be optional.